lib/Common/Memory/PageAllocator.cpp

```diff
@@ -124,6 +124,8 @@ SegmentBase<T>::Initialize(DWORD allocFlags, bool excludeGuardPages)
         return false;
     }
 
+    Assert( ((ULONG_PTR)this->address % (64 * 1024)) == 0 );
```

Comment:
Isn't `AutoSystemInfo::Data.dwAllocationGranularity` equal to pageSize ? == 4096?